puzzles/forms.py

```diff
@@ -125,34 +126,34 @@ def clean(self):
             .exists()
         ):
             raise forms.ValidationError(
-                'One of the emails you listed is already on a different team.'
+                _('One of the emails you listed is already on a different team.')
             )
 
 
 class SubmitAnswerForm(forms.Form):
     answer = forms.CharField(
-        label='Enter your guess:',
+        label=_('Enter your guess:'),
         max_length=500,
     )
 
 
 class RequestHintForm(forms.Form):
     hint_question = forms.CharField(
         label=(
-            'Describe everything you\u2019ve tried on this puzzle. We will '
-            'provide a hint to help you move forward. The more detail you '
-            'provide, the less likely it is that we\u2019ll tell you '
-            'something you already know.'
+            _(u'Describe everything you\u2019ve tried on this puzzle. We will '
+            u'provide a hint to help you move forward. The more detail you '
+            u'provide, the less likely it is that we\u2019ll tell you '
+            u'something you already know.')
```

Comment:
It's true for python 3 but the problem here is with django-admin makemessages when it parses the files to look for strings to translate: it generates the output file with  strings by escaping the escaping characters if any, so `you\u2019ve tried` becomes `you\\u2019ve` in the translation file and the translate function can't find the translation because the key doesn't match with that extra `\`. It's probably a bug in django-admin but a workaround is either to write the actual unicode character in the source string, or prepend the string with u. In that case, the generated string to translate becomes `you’ve tried` in the translation file and there's no issue. It's also more readable for translators who don't necessarily know what escaped characters are.

Reply:
Ah, I see. In that case, IMO it would be clearer to use the actual Unicode character (and maybe leave a warning in the README?). You don't have to go and change all the strings, but if you wanted to it would be appreciated.

Reply:
Done, there weren't that many.